README.md

```diff
@@ -78,59 +78,60 @@ will produce the following object (`result` variable):
 
 ```json
 {
-  "ast": {
-    "_version": "3.0",
-    "metadata": [],
-    "name": "",
-    "description": "",
-    "resourceGroups": [
-      {
-        "name": "",
-        "description": "",
-        "resources": [
-          {
-            "name": "",
-            "description": "",
-            "uriTemplate": "/1",
-            "model": {},
-            "parameters": [],
-            "actions": [
-              {
-                "name": "",
-                "description": "",
-                "method": "GET",
-                "parameters": [],
-                "examples": [
-                  {
-                    "name": "",
-                    "description": "",
-                    "requests": [],
-                    "responses": [
-                      {
-                        "name": "200",
-                        "description": "",
-                        "headers": [],
-                        "body": "",
-                        "schema": ""
-                      }
-                    ]
-                  }
-                ]
-              }
-            ]
-          }
-        ]
-      }
-    ]
+  "element": "category",
```

Comment:
Oh good point, I guess I need to update drafter locally as well before I run it!